### DISCUSSION TITLE
TS2769 : [@typescript/electron] Argument of type '"wheel-event"' is not assignable to parameter of type '"will-resize"'

### DISCUSSION THREAD

**Discussion by raphael10-collab** [DefinitelyTyped/DefinitelyTyped#66156] [Issues with a @types package]
I'm getting this error : 

    ERROR in ./src/main/index.ts:404:25
    TS2769: No overload matches this call.
      The last overload gave the following error.
        Argument of type '"wheel-event"' is not assignable to parameter of type '"will-resize"'.
        402 |         //})
        403 |
      > 404 |         SearchWindow.on("wheel-event", (event, args) => {
            |                         ^^^^^^^^^^^^^
        405 |             console.log("SearchWindow.on-wheel-event-args: ", args)
        406 |         })

In `main` process : 

        SearchWindow.on("wheel-event", (event, args) => {
            console.log("SearchWindow.on-wheel-event-args: ", args)
        })

In the `renderer` process: 

        let propertiesToClone = ['deltaX', 'deltaY', 'metaKey', 'ctrlKey', 'defaultPrevented', 'clientX', 'clientY']
        function cloneEvent (e) {
            let obj = {}
            for (let i = 0; i < propertiesToClone.length; i++) {
                obj[propertiesToClone[i]] = e[propertiesToClone[i]]
            }
            return JSON.stringify(obj)
        }

        setTimeout(function () {
            window.addEventListener('wheel', function (e) {
                window.api.electronIpcSend('wheel-event', cloneEvent(e))
            })
            let scrollTimeout = null
            window.addEventListener('scroll', function () {
                clearTimeout(scrollTimeout)
                scrollTimeout = setTimeout(function () {
                    window.api.electronIpcSend('scroll-position-change', Math.round(window.scrollY))
                }, 200)
            })
        }, 0)

People in the Electron Discord Forum say it's a typescript issue.

    electron: 25.3.1
    typescript: 4.5.4

Would you be so kind in helping find the root cause of the problem, in order to solve it?

[a comment by an automated account is not shown]

**Comment by codebytere** [comment]
@raphael10-collab Electron's types are built-in - they don't live here.

**Reply by raphael10-collab** [reply]
Hi1 @codebytere ! Does it mean that the issue in `ONLY` an `Electron` issue and should be solved only by `Electron`'s people ?

**Reply by codebytere** [reply]
@raphael10-collab yes - this can be closed.
